### PR TITLE
Fixed RViz2 linters

### PR DESCRIPTION
**Pull request by ahcorde**

### CODE REVIEW

File: rviz2/package.xml [hunk 1]
```diff
@@ -40,12 +40,9 @@
   <depend>rviz_ogre_vendor</depend>
 
   <!-- TODO(jacobperron): Replace with ament_lint_common when ament_copyright is working -->
```
Comment (clalancette) [inline rviz2/package.xml]: This TODO comment can be removed now.

**Review by clalancette — CHANGES_REQUESTED** [review]
One minor change, then this looks good.

File: rviz2/package.xml (hunk 1 above)
Comment (ahcorde) [reply rviz2/package.xml]: https://github.com/ros2/rviz/pull/1231/commits/b2d4d1b09772da1468065aca1b8320a9813c6d98

**Review by clalancette — APPROVED** [review]
Looks good to me with green CI.

**Comment by ahcorde** [comment]
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/21443/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/15731/)
* Linux-rhel [Build Status](http://ci.ros2.org/job/ci_linux-rhel/939/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/22163/)

**Pull request merged**